Dockerfile

```diff
@@ -31,6 +31,8 @@ RUN \
  install -m 0700 -o root -g root -d /root/config && \
  install -m 0700 -o root -g root -d /root/secrets
 
+RUN touch /var/lock/subsys/gratia-probes-cron
```

Comment:
That'll do!

Note i think `service gratia-probes-cron start` accomplishes the same thing, but yeah touching the file should be all you need to do to enable the probes service.  Of course the individual probe needs to be enabled in its ProbeConfig also.

Reply:
We don't have init in the container so we can't run `service...` D:

Reply:
Ah, makes sense.  Touching the file is simpler anyway.

(Not super relevant here, but in case it ever comes up that you want to run the `service` command, note that it actually comes with the `initscripts` package, which you can actually install independently of an init system.  `service` runs init scripts, but it does not require sysv init to be present.  init's job is mainly starting/stopping groups of services at system boot/shutdown, but supervisord takes care of that in the osg containers.)